### PR TITLE
feat(vscode): add icon to package

### PR DESCRIPTION
**Pull request by hugo-vrijswijk**
image

### CODE REVIEW

**Comment by nicojs** [comment]
Could you also add it to the lit-xml's readme and monorepo readme?

**Comment by hugo-vrijswijk** [comment]
Added the images, but in the inline preview it doesn't show because the image is not on the master branch

File: packages/lit-xml/readme.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Lit XML 🔥
 
+<p align="center">
+  <img width="200" height="200" src="../vscode-lit-xml/images/icon.png">
```
Comment (nicojs) [inline packages/lit-xml/readme.md:4]: Please use absolute src path, so it will also work for NPM.

File: packages/lit-xml/readme.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Lit XML 🔥
 
+<p align="center">
```
Comment (nicojs) [inline packages/lit-xml/readme.md:3]: Align center? Sure this is correct html?

Comment (hugo-vrijswijk) [reply packages/lit-xml/readme.md:3]: Yup, here is Bill Murray to explain it: https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d#center-alignment-example

**Pull request merged**